controllers/apply-next/lib/validators.js

```diff
@@ -18,6 +18,21 @@ function multiChoice(options) {
         .single();
 }
 
+/**
+ * @see https://github.com/hapijs/joi/issues/622
+ */
+function conditionalOnMultiChoice({ ref, match, schema }) {
```

Comment:
The previous logic for these conditional multi-choice fields wasn't validating correctly. Based on the github issue linked here this is the way to do this. A little verbose so I added a wrapper function.